graylog2-web-interface/src/views/components/MigrateFieldCharts.jsx

```diff
@@ -186,11 +188,15 @@ const _onCancel = (setMigrationFinished) => {
 };
 
 const MigrateFieldCharts = () => {
+  const viewType = useContext(ViewTypeContext);
+  if (viewType !== View.Type.Search) {
+    return null;
+  }
```

Comment:
I thought about using the `<IfSearch>` component, but I want to avoid calling the localstorage on every dashboard page.